techniques/general/G183.html

```diff
@@ -24,7 +24,7 @@ <h3>Colors that would provide 3:1 contrast with black words and 4.5:1 contrast w
       </section>
       <section class="example">
          
-            <p>The hypertext links in a document are medium-light blue (#3366CC) and the regular text is black (#000000). Because the blue text is light enough, it has a contrast of 3.9:1 with the surrounding text and can be identified as being different than the surrounding text by people with all types of color blindness, including those individuals who cannot see color at all.</p>
+            <p>The hypertext links in a document are medium-light blue (<code>#3366CC</code>) and the regular text is black (<code>#000000</code>). Beyond the difference in color, the links have no other styles differences compared with the regular text. Because the blue text is light enough, it has a contrast of 3.9:1 with the surrounding text and can be identified as being different from the surrounding text by people with all types of color blindness, including those individuals who cannot see color at all. In addition, the links have <code>:focus</code> and <code>:hover</code> styles that reintroduce the underline when the links receive keyboard focus or when a mouse pointer hovers over them.</p>
```

Comment:
```suggestion
            <p>The hypertext links in a document are medium-light blue (<code>#3366CC</code>) and the regular text is black (<code>#000000</code>). Beyond the difference in color, the links have no other styles differences compared with the regular text. Because the blue text is light enough, it has a contrast of 3.9:1 with the surrounding text and can be identified as being different from the surrounding text by people with all types of color blindness, including those individuals who cannot see color at all. In addition to the contrast difference, the links have <code>:focus</code> and <code>:hover</code> styles that reintroduce the underline when the links receive keyboard focus or when a mouse pointer hovers over them.</p>
```